uni/lib/view/schedule/schedule.dart

```diff
@@ -147,9 +147,9 @@ class SchedulePageViewState extends GeneralPageViewState<SchedulePageView>
     RequestStatus scheduleStatus,
   ) {
     final tabBarViewContent = <Widget>[];
-    for (var i = 0; i < 5; i++) {
+    for (var day = 0; day < 6; day++) {
       tabBarViewContent
-          .add(createScheduleByDay(context, i, lectures, scheduleStatus));
+          .add(createScheduleByDay(context, day, lectures, scheduleStatus));
     }
     return tabBarViewContent;
   }
```

Comment:
You can:
- Inline this logic (it's small and not reusable)
- Make this declarative (e.g. `workDays.map((day) => createSchedule...))`